Alerting: Support WeCom as a contact point type (#40975)
grafana/grafana: docs/sources/alerting/old-alerting/notifications.md

Grafana alert notifications can be sent to [Sensu](<(https://sensu.io)>) Go as events via the API. This operation requires an API key. For information on creating this key, refer to [Sensu Go documentation](https://docs.sensu.io/sensu-go/latest/operations/control-access/use-apikeys/#api-key-authentication).

> ### WeCom

Reviewer: With the changes suggested, we need to move this whole section to the `contact-points.md` file since we're not implementing WeCom in the old alerting just the new one.

> To set up WeCom, you must create a WeCom robot webhook. In WeCom PC Client:

Reviewer: ```suggestion
WeCom contact points need a Webhook URL. These are obtained by setting up a WeCom robot on the corresponding group chat. To obtain a Webhook URL using the WeCom desktop Client please follow these steps:

  In WeCom PC Client:
```

> 1. Click the "..." in the top right corner of the chat group that you wanted to add the robot

Reviewer: ```suggestion
1. Click the "..." in the top right corner of a group chat that you want your alerts to be delivered to
```

2. Click "Add Group Robot", select "New Robot" and give your robot a name. Click "Add Robot"

> 3. There should be a Webhook URL in the panel. Click "Copy URL" and paste it in Grafana WeCom setting page and click "finish"

Reviewer: ```suggestion
3. There should be a Webhook URL in the panel.
```
